__tests__/__snapshots__/dashboard-converter.test.js.snap

```diff
@@ -65,6 +65,8 @@ title = \\"\\"
 title_size = \\"16\\"
 title_align = \\"left\\"
 show_legend = false
```

Comment:
for consistency I would put `show_legend = true`

Reply:
👍  I realized I didn't need to include a new graph as well. You can have the `legend_` properties with `show_legend: false` so I've left one as false and one as true :)